### DISCUSSION TITLE
WebpackError: TypeError: realFetch.call is not a function

### DISCUSSION THREAD

**Discussion by ppatel31** [gatsbyjs/gatsby#30478] [Help]
All the module used fetch have this issue

WebpackError: TypeError: realFetch.call is not a function

Describe the issue that you're seeing.

just use fetch like isomorphic-fetch

Clear steps describing how to reproduce the issue. **Please** link to a reproduction, this makes your issue _much_ easier to diagnose (seriously).

How to Make a Minimal Reproduction: https://www.gatsbyjs.com/contributing/how-to-make-a-reproducible-test-case/

realFetch.call not working

What should happen?

realFetch.call not working

What happened.

System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.13.0 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 89.0.4389.90
    Edge: 89.0.774.57
    Firefox: 86.0
    Safari: 14.0.3
  npmPackages:
    gatsby: ^3.1.2 => 3.1.2
    gatsby-plugin-canonical-urls: ^3.1.0 => 3.1.0
    gatsby-plugin-google-tagmanager: ^3.1.0 => 3.1.0
    gatsby-plugin-preconnect: ^1.2.0 => 1.2.0
    gatsby-plugin-react-helmet: ^4.1.0 => 4.1.0
    gatsby-plugin-remove-trailing-slashes: ^3.1.0 => 3.1.0
    gatsby-plugin-robots-txt: ^1.5.5 => 1.5.5
    gatsby-plugin-sitemap: ^3.1.0 => 3.1.0
    gatsby-plugin-verify-bing: ^1.1.26 => 1.1.27
    gatsby-remark-external-links: 0.0.4 => 0.0.4
    gatsby-source-filesystem: ^3.1.0 => 3.1.0
    gatsby-source-graphql: ^3.1.0 => 3.1.0
    gatsby-transformer-json: ^3.1.0 => 3.1.0
    gatsby-transformer-remark: ^3.1.0 => 3.1.0
  npmGlobalPackages:
    gatsby-cli: 3.1.0
    gatsby: 2.19.41

Run `gatsby info --clipboard` in your project directory and paste the output here.

**Comment by LekoArts** [comment]
Please see https://github.com/lquixada/cross-fetch/issues/15 & https://github.com/prisma-labs/graphql-request/issues/85 - this doesn't seem to be an issue with Gatsby
